shim.go

```diff
@@ -111,8 +111,8 @@ func (s *shim) forwardAllSignals() chan os.Signal {
 	return sigc
 }
 
-func (s *shim) resizeTty() error {
-	ws, err := term.GetWinsize(os.Stdin.Fd())
+func (s *shim) resizeTty(tty *os.File) error {
```

Comment:
I sounds like it do resizeTty on the tty(argument).
It would be better to change the argument tty to fromTty.